apps/mobile/src/components/qrcode/Scan.tsx

```diff
@@ -47,7 +46,7 @@ export const CodeScan = () => {
     }
 
     // to disable camera when not in use
-    if (!isFocused()) return null
+    // if (!isFocused) return null
```

Comment:
If you remove this line the camera stays running in the background even if the user switches to another screen, overheating the phone and keeping the camera in use.

Please uncomment this line